asrtoolkit/alignment/align.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+import io
+import json
+import logging
+import os
+import pickle
+from os.path import basename, dirname, exists
+from os.path import join as join_path
+from pathlib import Path
+
+import en_core_web_sm
+
+# Third Party
+import spacy
+from spacy.tokens import Doc as spacy_doc
+from textacy.extract import ngrams, noun_chunks
+from toolz.sandbox.core import pluck
+
+from asrtoolkit.alignment.align_utils import (
+    AlignedDoc,
+    Extractor,
+    dict_to_segments,
+    word_lattice_to_lines,
+)
+from asrtoolkit.alignment.initialize_logger import initialize_logger
+
+initialize_logger()
```

Comment:
This shouldn't be needed since `align_utils` initialized the logger. I believe the logger `basicConfig` settings are global.